lib/Component.lua

```diff
@@ -424,4 +431,33 @@ function Component:_unmount()
 	self._handle = nil
 end
 
+--[[
+	Performs property validation, if it is enabled.
+]]
+function Component:_validateProps(props)
+	if not GlobalConfig.getValue("propertyValidation") then return end
```

Comment:
Let's keep these types blocks to one statement per line, which I think makes them a little easier to read with how keyword-noisy Lua is:

```lua
if not GlobalConfig.getValue("propertyValidation") then
    return
end
```